automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/notebooks/NotebookClusterMonitoringSpec.scala

```diff
@@ -102,7 +103,7 @@ class ClusterMonitoringSpec extends FreeSpec with LeonardoTestUtils with Paralle
 
           withNewCluster(project, request = request) { cluster =>
             withWebDriver { implicit driver =>
-              withNewNotebook(cluster, PySpark3) { notebookPage =>
+              withNewNotebook(cluster, notebooks.PySpark3) { notebookPage =>
```

Comment:
Just `PySpark3` should work here.

Same for other places where we reference `NotebookKernel` in this spec